packages/slate-react/src/components/content.js

```diff
@@ -270,6 +270,11 @@ class Content extends React.Component {
     let el
 
     try {
+      if (target.nodeType === 8) {
+        // on Firefox, sometimes the node can be comment which doesn't have .closest and it crashes
```

Comment:
Hey @k7d, can you copy the `COMPAT:` style comments from elsewhere in the codebase?

Reply:
@ianstormtaylor sure thing. done